app/src/main/java/org/wikipedia/suggestededits/Contribution.kt

```diff
@@ -3,8 +3,8 @@ package org.wikipedia.suggestededits
 import org.wikipedia.dataclient.WikiSite
 import java.util.*
 
-class Contribution internal constructor(val qNumber: String, var title: String, var description: String, val editType: Int, var imageUrl: String,
-                                        val date: Date, val wikiSite: WikiSite, var pageViews: Long, var tagCount: Int) {
+class Contribution internal constructor(val qNumber: String, var title: String, var description: String, val editType: Int, var imageUrl: String?,
+                                        val date: Date, val wikiSite: WikiSite, var pageViews: Long) {
```

Comment:
tagCount was added to assist in the dependent PR for displaying each contribution details

Reply:
Since each edit can add only one tag at most, how would this be helpful?

Reply:
This details design also makes more sense when grouped... if it is agreed to not group then its ok.. we don't need to 
![tags](https://user-images.githubusercontent.com/9540770/84447229-571f5180-abfc-11ea-95ee-262ffcbccb43.png)